inject-testing-data/src/app/app.py

```diff
@@ -25,24 +25,24 @@ def say_hello(response: Response):
     return "This route is needed for the auth check not to fail."
 
 
-@app.get("/api/repository/{id}/files-content")
+@app.get("/inject/api/repository/{id}/files-content")
```

Comment:
I think it would be even more elegant to use an `APIRouter`:
```python
router = APIRouter(
    prefix="/inject/api"
)

...

@router.get("/repository/{id}/files-content")
...
```

Reply:
Yeah, good idea! I will do that!

Reply:
In that case we couldn't keep the route "/". Therefore, I will leave it like that for now. 
But what could be done in the future is to restructure the whole code, so that only the route stays in app.py and rest is pushed to a inject_testing_data_request.py or something similar.